qa/setup_packages.py

```diff
@@ -92,6 +100,21 @@ def get_alias(version):
         """
         return getattr(version, "alias", None)
 
+
+    def get_dependencies(self, cuda_version=None, idx=None):
+        """Obtains dependant packages list if exists. Otherwise return None
```

Comment:
This doc made me check, what happens if you pass `None` to `join`. Unsurprisingly, it fails :)

Reply:
I believe the doc is not accurate now. We are returning empty string, not None

Reply:
Fixed

Reply:
Fixed